include/SSVOpenHexagon/Core/MenuGame.hpp

```diff
@@ -26,6 +26,31 @@
 namespace hg
 {
 
+class HexagonDialogBox // can't be named DialogBox due to a pre-existing C++ class
```

Comment:
This is surprising. Where is this `DialogBox` coming from?

Reply:
I was half wrong in my statement, it is not a class but a MinGW define
```
#define DialogBox __MINGW_NAME_AW(DialogBox)
```
This is what pops up in my code editor